### PR TITLE
Save state

### PR DESCRIPTION
**Pull request by samikuikka**
button for saving the content panels

### CODE REVIEW

File: frontend/src/components/ContentPanel/ContentPanelHeader/index.tsx [hunk 1]
```diff
@@ -28,8 +30,11 @@ export const ContentPanelHeader: React.FC<ContentPanelHeaderProps> = ({
                         setCategory={setCategory}
                     />
                 </div>
-                <div className="flex-1 z-10 pl-8">
-                    <DropdownMenu addPromptBoxes={addPromptBoxes} />
+                <div className="flex-1 z-30 pl-8">
```
Comment (KorpelainenTommi) [inline frontend/src/components/ContentPanel/ContentPanelHeader/index.tsx]: Putting this at z-30 causes the dropdown to be rendered over the navigation bar.
Consider switching this to `z-[15]`

image

**Review by KorpelainenTommi — CHANGES_REQUESTED** [review]
Problem with z-index

File: frontend/src/components/ContentPanel/ContentPanelHeader/index.tsx (hunk 1 above)
Comment (samikuikka) [reply frontend/src/components/ContentPanel/ContentPanelHeader/index.tsx]: Good catch!

**Review by KorpelainenTommi — APPROVED** [review]
Approved after change

**Pull request merged**